app/models/rules/sealed_bid.rb

```diff
@@ -0,0 +1,53 @@
+module Rules
+  class SealedBid < Rules::Basic
+    def winning_bid
+      return Presenter::Bid::Null.new if auction.available?
```

Comment:
But I will admit I'm not entirely sure of the difference between `Presenter::Bid::Null` and `NullBid`